input_box_widget.py

```diff
@@ -0,0 +1,28 @@
+from PyQt5 import QtWidgets, QtCore
+from PyQt5.QtWidgets import QApplication, QMainWindow, QLineEdit, QLabel
+from PyQt5.QtGui import *
+import sys
+class Inputbox(QMainWindow):
```

Comment:
This input box should inherit from the QLineEdit class, not QMainWindow. Right now you're putting a QMainWindow (input box) into another QMainWIndow (Game).

```suggestion
class Inputbox(QtWidgets.QlineEdit):
```